### PR TITLE
[Bug-1339532] Creates the first pytest fixture to simulate its performance

### PR DESCRIPTION
**Pull request by andela-aolasode**
### What does this PR do?
* Configure pytest for the first test suite in test_db.py file.

### Other information
* This PR is raised to receive feedback.

### CODE REVIEW

File: auslib/test/test_db.py [hunk 1]
```diff
@@ -149,19 +151,30 @@ def testMultiplePotentialSignoffsForOneGroupWithEnoughSignoffs(self):
         verify_signoffs(required, signoffs)
 
 
+@pytest.fixture(scope='session')
+def create_db():
+    def the_db(theSelf):
+        MemoryDatabaseMixin.setUp(theSelf)
+        theSelf.engine = create_engine(theSelf.dburi)
+        return theSelf.engine
+    return the_db
```
Comment (andela-aolasode) [inline auslib/test/test_db.py]: Thanks, I will look into that.

File: auslib/test/test_db.py [hunk 2]
```diff
@@ -184,7 +197,8 @@ def testRollback(self):
     # bug 740360
     def testContextManagerClosesConnection(self):
         with AUSTransaction(self.metadata.bind.connect()) as trans:
-            self.assertEqual(trans.conn.closed, False, "Connection closed at start of transaction, expected it to be open.")
+            self.assertEqual(trans.conn.closed, False,
+                             "Connection closed at start of transaction, expected it to be open.")
```
Comment (andela-aolasode) [inline auslib/test/test_db.py]: Thank you. I will avoid mixing them together next time

File: auslib/test/test_db.py [hunk 3]
```diff
@@ -375,7 +389,8 @@ def testUpdateFailsOnVersionMismatch(self):
 
     def testUpdateClosesConnectionOnImplicitTransaction(self):
         with mock.patch('sqlalchemy.engine.base.Connection.close') as close:
-            self.test.update(changed_by='bob', where=[self.test.id == 1], what=dict(foo=432), old_data_version=1)
+            self.test.update(changed_by='bob', where=[
+                             self.test.id == 1], what=dict(foo=432), old_data_version=1)
```
Comment (andela-aolasode) [inline auslib/test/test_db.py]: okay, I will implement that.

**Comment by mozbhearsum** [comment]
@andela-aolasode doesn't have time to finish this up, so I'm closing it. It's still a good prototype for the future though. This work is being tracked in https://bugzilla.mozilla.org/show_bug.cgi?id=1339532.

**Pull request closed without merging**